mock/api.js

```diff
@@ -151,40 +134,60 @@ export const mockSamples = {
 };
 
 export const mockDetail = {
-  hits: {
-    hits: [
-      {
-        _source: {
-          run: {
-            id: 'test_id',
-            controller: 'test_controller',
-            name: 'test_result',
-            script: casual.random_element(['fio', 'uperf']),
-            date: moment.utc(),
-            start: moment.utc(),
-            end: moment.utc(),
-            user: casual.username,
-          },
-          host_tools_info: [
-            {
-              hostname: 'test_host',
-              tools: {
-                iostat: '--interval=3',
-                mpstat: '--interval=3',
-                perf: "--record-opts='record -a --freq=100'",
-                pidstat: '--interval=30',
-                'proc-interrupts': '--interval=3',
-                'proc-vmstat': '--interval=3',
-                sar: '--interval=3',
-                turbostat: '--interval=3',
-              },
-            },
-          ],
-        },
-      },
-    ],
-  },
+  hostTools: new Array(DEFAULT_SIZE).fill().map(() => ({
```

Comment:
There are two keys in the JSON doc returned by `/datasets/detail`: `"hostTools"` and `"runMetadata"`.  You're only returning the `hostTools`.

```
  "runMetadata": {
    "config": "test-03",
    "controller": "gprfc056.sbu.lab.eng.bos.redhat.com",
    "controller_dir": "gprfc056.sbu.lab.eng.bos.redhat.com",
    "date": "2021-05-24T03:27:21",
    "end": "2021-05-24T03:28:29.949026",
    "file-date": "2021-05-24T03:28:25.292000",
    "file-name": "/srv/pbench/archive/fs-version-001/gprfc056.sbu.lab.eng.bos.redhat.com/pbench-user-benchmark_test-03_2021.05.24T03.27.21.tar.xz",
    "file-size": 5944316,
    "id": "3646953262b0adfe6d1c32903912087b",
    "iterations": "1-default",
    "md5": "3646953262b0adfe6d1c32903912087b",
    "name": "pbench-user-benchmark_test-03_2021.05.24T03.27.21",
    "pbench-agent-version": "v0.71.0-185g5527f23f1",
    "raw_size": 6828923,
    "script": "pbench-user-benchmark",
    "start": "2021-05-24T03:27:55.061895",
    "tar-ball-creation-timestamp": "2021-05-24T03:28:38.255317",
    "toc-prefix": "pbench-user-benchmark_test-03_2021.05.24T03.27.21",
    "toolsgroup": "default"
  }
```

Reply:
fixed!